### PR TITLE
[WIP]Custom Vitro java classes

### PR DESCRIPTION
**Pull request by jgreben**
Adds framework support for custom Vitro java classes, and a basic TDB loading main class that loads a single record. It currently throws a TDB error which is expected based on prior experimentation with the command-line tdbloader.

### CODE REVIEW

**Comment by jcoyne** [comment]
What ticket is this work in support of?

**Comment by mjgiarlo** [comment]
@jcoyne Refs sul-dlss/rialto#73, sul-dlss/rialto#63, sul-dlss/rialto#62

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,5 @@ utilities/sdb_to_tdb/.work
 **/target
 
 **/overlays
+
+/rialto/lib
```
Comment (jcoyne) [inline .gitignore:21]: Why is this ignored?

Comment (jcoyne) [reply .gitignore:21]: Ah, is this the build directory?

**Comment by jcoyne** [comment]
Should this be moved to https://github.com/sul-dlss/rialto?

**Comment by mjgiarlo** [comment]
@jcoyne @jgreben @tingulfsen Can y'all combine your powers, superfriends-like, to take this PR to the next stage? Thanks!

**Comment by jcoyne** [comment]
@mjgiarlo I have no idea what that means.

**Comment by mjgiarlo** [comment]
@jcoyne Super Friends

**Comment by cmharlow** [comment]
closing PR due to practice for now of keeping assessment work in own branches.

**Pull request closed without merging**